### PR TITLE
run rustfmt on librustc_back

### PR DESCRIPTION
**Pull request by srinivasreddy**

### CODE REVIEW

File: src/librustc_back/dynamic_lib.rs [hunk 1]
```diff
@@ -39,23 +37,26 @@ impl DynamicLibrary {
...
         }
     }
 
     /// Prepends a path to this process's search path for dynamic libraries
     pub fn prepend_search_path(path: &Path) {
         let mut search_path = DynamicLibrary::search_path();
         search_path.insert(0, path.to_path_buf());
-        env::set_var(DynamicLibrary::envvar(), &DynamicLibrary::create_path(&search_path));
+        env::set_var(DynamicLibrary::envvar(),
+                     &DynamicLibrary::create_path(&search_path));
```
Comment (GuillaumeGomez) [inline src/librustc_back/dynamic_lib.rs]: I'm not sure to understand why this change has been done (it's under 100 columns after all).

File: src/librustc_back/dynamic_lib.rs [hunk 2]
```diff
@@ -74,7 +75,11 @@ impl DynamicLibrary {
     }
 
     fn separator() -> &'static str {
-        if cfg!(windows) { ";" } else { ":" }
```
Comment (GuillaumeGomez) [inline src/librustc_back/dynamic_lib.rs]: Once again, not sure why this change. Strange...

File: src/librustc_back/target/apple_ios_base.rs [hunk 3]
```diff
@@ -79,7 +79,8 @@ fn target_cpu(arch: Arch) -> String {
         Arm64 => "cyclone",
         I386 => "yonah",
         X86_64 => "core2",
-    }.to_string()
+    }
+    .to_string()
```
Comment (GuillaumeGomez) [inline src/librustc_back/target/apple_ios_base.rs]: Really don't like this one. :-/

File: src/librustc_back/dynamic_lib.rs (hunk 2 above)
Comment (srinivasreddy) [inline src/librustc_back/dynamic_lib.rs]: if expressions are expanded.

File: src/librustc_back/target/apple_ios_base.rs (hunk 3 above)
Comment (srinivasreddy) [inline src/librustc_back/target/apple_ios_base.rs]: lets see what @nrc says

File: src/librustc_back/dynamic_lib.rs (hunk 1 above)
Comment (srinivasreddy) [inline src/librustc_back/dynamic_lib.rs]: yep. sometimes multi line method call has been turned to single line. and vice versa. Need to find the reason.

File: src/librustc_back/target/apple_ios_base.rs (hunk 3 above)
Comment (nrc) [inline src/librustc_back/target/apple_ios_base.rs]: This is one of the more common complaints at the moment. It's not ideal, but it's not likely to get fixed in the very short term (it's sub-optimal, but I don't think it is wrong or super-ugly and isn't a bug). I'd suck this up for now, on the understanding that we'll get to it at some point in the near-ish future and it will get prettier.

(btw, the reason it's not trivial to fix is that if there is more than one method call or the call is more complicated it can be desirable to do it this way, then consistency comes into play).

**Comment by srinivasreddy** [comment]
@nrc rollup or close this branch ?

**Comment by alexcrichton** [comment]
Closing due to inactivity, but feel free to resubmit with comments addressed!
